AVFoundation-CombineTests/TestSubscriber.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  TestSubscriber.swift
+//  AVFoundation-CombineTests
+//
+//  Created by József Vesza on 2020. 07. 24..
+//  Copyright © 2020. József Vesza. All rights reserved.
+//
+
+import Combine
+
+class TestSubscriber<T>: Subscriber {
+    typealias Input = T
+    typealias Failure = Never
+    
+    let demand: Int
+    let onComplete: ([T]) -> Void
+    
+    private var receivedValues: [T] = []
```

Comment:
```suggestion
    /// An Array to store the values received from a `Subscription`.
    private var receivedValues: [T] = []
```

Reply:
Added a slightly tweaked version of this